### PR TITLE
Better localStore

### PR DESCRIPTION
**Pull request by N8python**
# Pull Request

### CODE REVIEW

File: masmas.js [hunk 1]
```diff
@@ -85,6 +85,8 @@ function localStore(varname, val) {
     window[varname] = Number(localStorage[varname]);
   } else if (typeof val === 'boolean') {
     window[varname] = Boolean(localStorage[varname]);
+  } else if (typeof val === 'object'){
```
Comment (RDIL) [inline masmas.js]:
```suggestion
  } else if (typeof val === 'object') {
```

**Comment by RDIL** [comment]
Nate, when I request changes you can't just merge the pull request.

**Comment by N8python** [comment]
What changes do you want?

**Comment by RDIL** [comment]
Scroll up and look

**Comment by N8python** [comment]
I don’t see any changes?

**Comment by RDIL** [comment]
@N8python 
https://rdil.rocks/depression/Screen%20Shot%202019-01-28%20at%204.50.21%20PM.png)

**Comment by N8python** [comment]
I saw that - but you didn’t change anything!

**Comment by RDIL** [comment]
@N8python that was me requesting you make changes, now I put branch protection rules in place.

**Pull request merged**